content/blog/2010/2010-08-02-dogfooding-hudson-were-looking-for-slaves.html

```diff
@@ -1,6 +1,6 @@
 ---
```

Comment:
Is it possible for us to change the title/filename and have the old location redirect?